policy-engine/src/graph.rs

```diff
@@ -144,9 +147,30 @@ where
     let graph_json = serde_json::to_string(&internal_io.graph)
         .map_err(|e| GraphError::FailedJsonOut(e.to_string()))?;
 
+    let v =
+        serde_json::from_str(&graph_json).map_err(|e| GraphError::FailedJsonOut(e.to_string()))?;
+    let version: i32 = match internal_io.parameters.get("version") {
+        None => *commons::MIN_CINCI_VERSION as i32,
+        Some(v) => v.parse::<i32>().unwrap(),
+    };
+
+    let updated_json = add_version_information(&v, &version);
+
     Ok(HttpResponse::Ok()
         .content_type(CONTENT_TYPE)
```

Comment:
This should be consuming the version from `get_version` (possibly by looking in `plugin_params`).  So `get_version` might need to return both a version integer and a recommended media type string, so we can satisfy folks who ask for v1 with `application/json` and folks who ask for that same content via `application/vnd.redhat.cincinnati.v1+json`.